src/nfa/nfa.cc

```diff
@@ -781,30 +781,31 @@ std::pair<Run, bool> Mata::Nfa::get_word_for_path(const Nfa& aut, const Run& run
 
 bool Mata::Nfa::is_in_lang(const Nfa& aut, const Run& run)
 {
-    StateSet cur = aut.initial_states;
+    //StateSet cur = aut.initial;
+    StateSet cur(aut.initial);
 
     for (Symbol sym : run.word)
     {
         cur = aut.post(cur, sym);
         if (cur.empty()) { return false; }
     }
 
-    return !are_disjoint(cur, aut.final_states);
+    return !are_disjoint(cur, aut.final);
 }
 
 /// Checks whether the prefix of a string is in the language of an automaton
 bool Mata::Nfa::is_prfx_in_lang(const Nfa& aut, const Run& run)
 {
-    StateSet cur = aut.initial_states;
+    StateSet cur = (StateSet(aut.initial));
```

Comment:
Again, mentally check if this is correct and you are initializing from elements and not bools.

Reply:
yes yes, I added that contructor ot OrdVector